### ISSUE TITLE
[Feature] E-Ink Support? 

### ISSUE THREAD

**Issue by jeffery9876** [FreshRSS/FreshRSS#4396]
**Is your feature request related to a problem? Please describe.**
<br>
I have been searching for solution to read FreshRSS feed in my android E-Ink reader. all the [android app listed in here](https://github.com/FreshRSS/FreshRSS#apis--native-apps) are relying on scrolling view, which leads to very bad experience on the E-ink devices.   
<br>
So I opened the FreshRSS from browser, and it is still facing same problem.

---
**Describe the solution you’d like**

I think the solution is simple, adding next/previous page button. This website, [Reabble](url), as an example has optimised for E-ink reading, but it only supports connecting Inoreader. 

https://reabble.com/session/new/demo

--------

My question is: 
- Is there any existing android app supporting FreshRSS and E-ink?
- Is it easy to have a theme on hosted FreshRSS to supporting E-ink? 

Thank you very much

**Comment by Frenzie** [comment]
I don't know if there are e ink optimized browsers, but Dolphin for example has an option to stick page up/down buttons on the screen iirc.

You could do the same thing with a FreshRSS extension or custom JS but having it in the browser is presumably a lot better.

**Comment by Alkarex** [comment]
Hello,
At the moment, we have some buttons, for instance to go the next article without scrolling, but what you are asking is mainly to move down into a long article, right?

**Comment by Frenzie** [comment]
Yes, basically a button that does something like this and vice versa:

```js
scrollTo(scrollX, scrollY + innerHeight - innerHeight/10)
```

Like I said, I think it makes more sense to do this at the browser level, but it's otherwise fairly trivial.

**Comment by jeffery9876** [comment]
Hi Guys.

Someone pointed out that Feedme App has E-Ink mode, and with button control for page up/down. I will give it a try and revert back

**Comment by foureight84** [comment]
This would be a wonderful feature to have. Perhaps just a simple URL param to toggle the nav menu items to turn on signage mode? I have a Visonect e-ink device, better known as the Joan e-ink display, and I am currently using one with FreshRSS to display the latest news headlines.

My current solution, as suggested by other users, is to use the "Custom CSS" extension to target the device resolution to hide nav menu items. It works for now.

**Comment by rezad1393** [comment]
is there any way to at least add a e-ink optimized theme?
right now there is no pure white theme.
feed entry in original theme have a mild orange tint.
I understand that is for differentiating the title from feed text but on e-ink it messes with the quality of display.

I have a color e-ink am willing test themes proposed for e-ink.

**Comment by Alkarex** [comment]
> is there any way to at least add a e-ink optimized theme?

Yes, that is easy to do, especially since most of our themes have colours declared as variables, so taking an existing theme and tuning it should not take that long. Help and PR welcome

**Comment by rezad1393** [comment]
> > is there any way to at least add a e-ink optimized theme?
> 
> Yes, that is easy to do, especially since most of our themes have colours declared as variables, so taking an existing theme and tuning it should not take that long. Help and PR welcome

I am too much of a noob but I will try to make one.
